### PR TITLE
Patch harfbuzz's cmake platform check

### PR DESCRIPTION
**Pull request by fish47**

### CODE REVIEW

**Comment by fish47** [comment]
https://github.com/vitasdk/packages/issues/300

**Comment by fish47** [comment]
I don't know whether in-source path is acceptable or not, but it's convenient and straightforward, so I move the patch content to the buildscript.

**Comment by sharkwouter** [comment]
Hey @fish47, thanks for making this! I do think putting the patch in a separate file is a bit more readable.

**Comment by fish47** [comment]
Ok, just follow the code convention. Should I submit a new commit on this branch, or make the final commit and force push it on this branch, or create another branch and pull request?

**Comment by sharkwouter** [comment]
I would remake the commit and force push, it leads to a cleaner history in git. We can keep it in this PR.

**Review by sharkwouter — APPROVED** [review]

File: harfbuzz/VITABUILD [hunk 1]
```diff
@@ -2,8 +2,19 @@ pkgname=harfbuzz
 pkgver=8.2.2
 pkgrel=1
```
Comment (sharkwouter) [inline harfbuzz/VITABUILD:3]: This has to be increased

**Review by sharkwouter — CHANGES_REQUESTED** [review]

**Comment by sharkwouter** [comment]
Thanks for updating and sorry for changing my mind on the approval. The pkgrel should be updated, otherwise this looks good.

**Comment by fish47** [comment]
Done. Thanks for your adivce. Inspecting any potential bugs or subtle mistakes before release is what code review should be.

**Review by sharkwouter — APPROVED** [review]

**Comment by sharkwouter** [comment]
Thanks for updating. I've approved the change now. Now we'll just need someone with the right permissions to click merge.

**Pull request merged**